src/lib/ui/containers/SignupForm/SignupForm.svelte

```diff
@@ -75,6 +77,43 @@
       success,
     }
   );
+
+  $: fields = (
+    [
+      {
+        name: "firstName",
+        type: "text",
+      },
+      {
+        name: "lastName",
+        type: "text",
+      },
+      {
+        name: code === "bycountry" ? "country" : "phoneCode",
+        type: "select",
+      },
+      {
+        name: "email",
+        type: "email",
+      },
+      {
+        name: "phone",
+        type: "tel",
+      },
+      {
+        name: "password",
+        type: "password",
+      },
+      ...(confirm
+        ? ([
+            {
```

Comment:
🚫 **[eslint]** <[@typescript-eslint/indent](https://typescript-eslint.io/rules/indent)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected indentation of 10 spaces but found 12.
```suggestion
          {
```